sdk/purview/purview-catalog-rest/test/public/utils/recordedClient.ts

```diff
@@ -24,42 +17,25 @@ const replaceableVariables: { [k: string]: string } = {
   AZURE_TENANT_ID: "88888888-8888-8888-8888-888888888888",
 };
 
-export function createClient(options?: ClientOptions): PurviewCatalogRestClient {
+const recorderOptions: RecorderStartOptions = {
+  envSetupForPlayback: replaceableVariables,
+};
+
+export async function createClient(
+  recorder: Recorder,
+  options?: ClientOptions
+): Promise<PurviewCatalogRestClient> {
   const httpClient = isNode || isLiveMode() ? undefined : createXhrHttpClient();
```

Comment:
The legacy recorder didn't support recording browser calls made through the Fetch API. As a result, when fetch API was introduced in `core-rest-pipeline` a workaround was applied to the libraries not yet migrated to the new recorder, this workaround needs to be removed as part of the migration. Please make the following changes

```diff
- const httpClient = isNode || isLiveMode() ? undefined : createXhrHttpClient();
```

```diff
- recorder.configureClientOptions({ httpClient, ...options })
+ recorder.configureClientOptions(options)
```

Reply:
Acknowledge!